supertokens/supertokens.go

```diff
@@ -54,16 +54,20 @@ func supertokensInit(config TypeInput) error {
 	}
 
 	if config.Supertokens != nil {
-		hostList := strings.Split(config.Supertokens.ConnectionURI, ";")
-		var hosts []NormalisedURLDomain
-		for _, h := range hostList {
-			host, err := NewNormalisedURLDomain(h)
-			if err != nil {
-				return err
+		if len(config.Supertokens.ConnectionURI) != 0 {
```

Comment:
You need to change changelog

Reply:
added relevant change message to the changelog file